controllers/articles-controller.js

```diff
@@ -19,6 +19,16 @@ function getArticles(req, res, next) {
 };
 
 
+function deleteComment(req, res, next) {
+    const { comment_id } = req.params;
+    removeComment(comment_id)
+    .then(() => {
+        res.status(204).send()
```

Comment:
A nice shorthand you can use here is `.sendStatus(204);` as there is nothing to send